app/templates/suppliers/become_a_supplier.html

```diff
@@ -0,0 +1,91 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Become a supplier – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+  {%
+    with heading = "Become a supplier",
+         smaller = True
+  %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      {% if open_fwks or opening_fwks %}
+        <div class="dmspeak">
+          <h2 class="heading-xmedium padding-bottom-small">Services you can apply to sell</h2>
+
+        {% for framework in open_fwks + opening_fwks %}
+
+          {{ content_loader.get_message(framework.slug, 'become-a-supplier', framework.status) }}
+          {% if not loop.last %}
+          <p>&nbsp;</p>
```

Comment:
?

Reply:
It looks shit without it...

Reply:
Shouldn't we have a "double-spaced paras" style then?

Reply:
Probably...

Reply:
Better...?
The gap using this style is slightly smaller but looks fine to me:
![screen shot 2017-09-15 at 16 59 58](https://user-images.githubusercontent.com/6525554/30492099-56a050b2-9a37-11e7-94a1-596a726d6e4a.png)

Reply:
I guess so... feels like something lacking in our stylesheets if we can't get a bulleted list displaying in a way we think "looks good" without wrapping things in extra elements...

Reply:
The tricky thing is that the bulleted list comes out of frameworks content, and the lists there don't know if they will be displayed in the middle or end of the page, hence the need to wrap them in the template here.
Otherwise we could just add the style to the list itself.